src/vm/tests/integrations.rs

```diff
@@ -168,9 +188,15 @@ fn integration_test_get_info() {
     let num_rounds = 4;
 
     let mut run_loop = testnet::helium::RunLoop::new(conf);
+
+    unsafe {
```

Comment:
Woah now, why does this need to be `unsafe`?

Reply:
It needs to modify a static variable, which is later read by the helium node passed function pointers. Those cannot be closures, so they can only read static variables.

Reply:
Can't you do this with an `Arc<T>`?

Reply:
I believe it would need to be a lazy_static Mutex wrapped option. I can update to use that.

Reply:
updated in 5951759a789563a6ecb1ae3800fc0706944b02ec